src/github3/models.py

```diff
@@ -259,7 +259,7 @@ def ratelimit_remaining(self):
 
         :returns: int
         """
-        json = self._json(self._get(self._github_url + '/rate_limit'), 200)
+        json = self._json(self._get(self._api + '/rate_limit'), 200)
```

Comment:
I don't think this is  correct because it leads to bad requests like https://travis-ci.org/sigmavirus24/github3.py/jobs/445400121#L584. I'm actually not sure what it should be instead...

Reply:
Rate limit hangs off the base API url, it's not off of any specific object. I was digging around yesterday to think about how to expose the GHE specific API link, and do it in a way here that is transparent, and I wasn't able to come up with anything quickly.

Reply:
Actually this should be `self._build_url('rate_limit')` not what we have here.